blog/templates/blog/post_list.html

```diff
@@ -0,0 +1,15 @@
+{% extends "blog/base.html" %}
+
+{% block content %}
+    {% for post in posts %}
+        <div class="post">
+            <div class="date">
+                <p>Опубликовано: {{ post.published_date }}</p>
```

Comment:
Please replace **all** translatable strings with i18n (`{% trans "Posted on:" %}`).
This requires `{% load i18n %}`.